net/rpc/server.go

```diff
@@ -495,7 +516,17 @@ func (server *Server) ServeRequest(codec ServerCodec) error {
 		}
 		return err
 	}
-	service.call(server, sending, nil, mtype, req, argv, replyv, codec)
+
+	handler :=  func() {
```

Comment:
I think you should run `gofmt` on the project; there may be more whitespace issues I haven't found
```suggestion
	handler := func() {
```